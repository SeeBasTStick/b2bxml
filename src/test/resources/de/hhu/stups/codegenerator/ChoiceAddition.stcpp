int main() {
    Choice choice;
    choice.calculate(BInteger(3));
    return 0;
}