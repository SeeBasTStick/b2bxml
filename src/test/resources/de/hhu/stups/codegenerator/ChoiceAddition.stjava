public static void main(String[] args) {
    Choice choice = new Choice();
    choice.calculate(new BInteger(3));
    System.out.println(choice.getRes());
}