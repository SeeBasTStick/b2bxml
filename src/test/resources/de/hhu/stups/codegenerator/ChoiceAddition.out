public static void main(String[] args) {
    Choice choice = new Choice();
    choice.Inc(new BInteger(10));
}